src/palette/view/widgets/palettewidget.cpp

```diff
@@ -609,10 +609,13 @@ QPixmap PaletteWidget::pixmapForCellAt(int paletteIdx) const
 
     painter.setPen(Pen(color));
 
-    PaletteCellIconEngine::PaintContext ctx;
+    notation::IEngravingItemIconEngine::PaintContext ctx;
     ctx.painter = &painter;
 
-    element->scanElements(&ctx, PaletteCellIconEngine::paintPaletteItem);
+    QColor elemColor = configuration()->elementsColor();
+    ctx.color = &elemColor;
+
+    element->scanElements(&ctx, engravingIconEngine()->getItemPainterFunction());
```

Comment:
This is not a very elegant way to paint the element... I would prefer it to look like this:

engravingIconEngine()->painElement(element);